cypress/e2e/base.spec.ts

```diff
@@ -10,4 +10,23 @@ describe('Open browser and check if login button exists.', () => {
   });
 });
 
+describe('Visit landing page and verify users exists', () => {
+  beforeEach(() => {
+    cy.intercept('/api/users', { fixture: 'users.json' }).as('getUsers');
```

Comment:
The `getUsers` alias isn't being used anywhere - we can probably remove

Reply:
Also, general note for anyone: we may not be able to intercept API calls that are called on the server (as a result of SSR) - definitely worth looking into whether it's possible.

If it's not easily implemented, I think it's fine if we just don't test that particular functionality as a part of this PR

Reply:
What's SSR